README.md

```diff
@@ -61,14 +61,25 @@ cd /src/actinia-tiling-plugin/
 make test
 ```
 
-## Small Example
+For debugging the test this might be helpful when a `waitAsyncStatusAssertHTTP` fails:
+```
+from flask.json import loads as json_loads
+resp_data = json_loads(rv.data)
+rv_user_id = resp_data["user_id"]
+rv_resource_id = resp_data["resource_id"]
+rv2 = self.server.get(URL_PREFIX + "/resources/%s/%s" % (rv_user_id, rv_resource_id), headers=self.user_auth_header)
+resp_data2 = json_loads(rv2.data)
+```
 
-### Grid Tiling Example
+## Small Example
 ```
 actinia_base_url=http://localhost:8088/api/v3
-mapset_url=${actinia_base_url}/locations/loc_25832/mapsets/hpda_tiling_user
+mapset_url=${actinia_base_url}/locations/loc_25832/mapsets/hpda_tiling_usermapset
```

Comment:
```suggestion
mapset_url=${actinia_base_url}/locations/loc_25832/mapsets/hpda_tiling_usermapset
```
Here, and elsewhere: I suggest to remove the "hpda_" prefixes which are unrelated to actinia_core.